pepper-apis/dsm-core/src/main/resources/liquibase/DDP-1205_CleanupDeletedOncHistoryEtc.sql

```diff
@@ -0,0 +1,116 @@
+-- this file will be run manually, not by liquibase, but it is here to track
+-- our approach to DDP-1205.
+
+-- insert deleted onc history details
+insert into deleted_object (original_table, original_primary_key, data)
```

Comment:
Maybe use systemdate to store into deleted_date if we think it helps in anyway.
Since all deletions are same date, maybe we can add to the ticket when we deleted, once we run this delete in prod

Reply:
Good catch, and this makes me think we should make `data`, `deleted_by`, and `deleted_date` non-nullable.  @pegahtah what do you think of making these fields required?